cfme/infrastructure/cluster.py

```diff
@@ -111,20 +111,25 @@ def exists(self):
         except sel.NoSuchElementException:
             return False
 
+    @property
+    def cluster_id(self):
+        """extracts cluster id for this cluster"""
+        sel.force_navigate('infrastructure_clusters')
```

Comment:
When I'd changed this to use database requests instead, I realized that I could do the same thru rest_api. So, I've replaced this code to use rest.